contracts/handlers/HandlerHelpers.sol

```diff
@@ -42,18 +42,6 @@ contract HandlerHelpers is IERCHandler {
         require(msg.sender == _bridgeAddress, "sender must be bridge contract");
     }
 
-    /**
```

Comment:
Can we rename this `HandlerHelpers` to something that indicates that this is a helper contract only for ERC handlers?